lib/util/git/index.ts

```diff
@@ -509,7 +500,7 @@ export async function getFile(
 ): Promise<string | null> {
   await syncGit();
```

Comment:
i think we can do the sync after exists check, so we don't fetch if we throw an error anyways

Reply:
I've removed this and a few other redundant branchExists checks